### DISCUSSION TITLE
Subscribe to The GitHub Insider

### DISCUSSION THREAD

**Discussion by tuves** [community/community#60761] [Announcements]
Discover tips, technical guides, and best practices in our monthly newsletter for developers. Whether you’re a beginner or a professional, learn how GitHub can boost your productivity and happiness.

### Explore things like: 

**Keyboard shortcuts**: From quick and dirty editing to finding your code quickly, discover simple shortcuts that can save you time and headspace.
 
**GitHub Copilot magic**: Code and function suggestions are just the start—learn about the hidden capabilities of your AI-powered companion. 

**Securing your code in seconds**: Security should speed you up, not slow you down. Discover how you can keep your code secure with minimal effort.

Plus, get exclusive access to GitHub events, swag promos, technical previews, and more.  

[**SIGN ME UP!**](https://resources.github.com/newsletter/)

### Looking for more GitHub goodness?
 

- Visit our [Changelog](https://github.blog/changelog/) for the latest ships, launches, and product updates.

- Explore our [Shop](https://www.thegithubshop.com/) to browse and purchase GitHub swag.

- Subscribe to our [LinkedIn newsletter](https://www.linkedin.com/newsletters/6958196028076429312/) for career development and inspiration.

**Comment by AOHhiroko** [comment]
Hi, Iam at a loss , and exactly I need the things you mention, branch deploy action, technical guide, and so on.
give me advice to 
![1_BsaiqRZGFfIRQ3BorZXH3A](https://github.com/community/community/assets/56747113/038b43b9-5164-45e4-9551-8a5e90dad1aa)
take the step

**Comment by AOHhiroko** [comment]
I even do not know how to use files in my pc. it should be in my" waves "account.. I need to open my personal account in Git Hub.

**Comment by sina1100** [comment]
سلام ، من کجام ، اینجا چجور اومدم ، ولی با اینحال میدونم وظیفه دارم تشکر کنم از شما 🙌🙏🌹

**Comment by reeveng** [comment]
Does github insider mention anything about the new homepage?

**Reply by reeveng** [reply]
if so, i'll subscribe, give me an update please :heart:

**Comment by xieyuheng** [comment]
Please bring back the old feed!

**Comment by xieyuheng** [comment]
We need old feed back:

https://github.com/orgs/community/discussions/66244

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
